pdns/dnsdistdist/docs/conf.py

```diff
@@ -92,7 +92,7 @@
 changelog_render_changeset = "https://github.com/PowerDNS/pdns/commit/%s"
 
 changelog_sections = ['New Features', 'Improvements', 'Bug Fixes', 'Removals']
-changelog_inner_tag_sort = ['Security', 'DNS over HTTPS', 'DNS over TLS', 'DNSCrypt', 'DNSTAP', 'Protobuf', 'Performance', 'Webserver', 'Metrics']
+changelog_inner_tag_sort = ['Security', 'DNS over QUIC', 'DNS over HTTPS', 'DNS over TLS', 'DNSCrypt', 'DNSTAP', 'Protobuf', 'Performance', 'Webserver', 'Metrics']
```

Comment:
not a comment but a question, assuming you know: what does this do? :sweat_smile:

Reply:
:D It is used by the doc generation process to group entries with identical tags together in the ChangeLog. This is not very useful, to be honest :)